mobile/src/components/interview/components/applications/components/interview-parameters/interview-parameters.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { FC } from 'react';
+
+import { PermissionKey } from '~/common/enums/enums';
+import {
+  InterviewsGetAllItemResponseDto,
+  InterviewsGetInterviewerResponseDto,
+  InterviewsUpdateRequestDto,
+  UserWithPermissions,
+} from '~/common/types/types';
+import { Dropdown, Text, View } from '~/components/common/common';
+import { Category } from '~/components/course/components/components';
```

Comment:
We can't use components from other modules, also on this screen category shouldn't have a border. So let's create a common Category component without border and use it for interviews/interview-application screens, and in course screen create local component like CourseCategory that will wrap Category and add border and onPress